### ISSUE TITLE
Rule Proposal: warn if tag name values are incorrect

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#59]
Sample use case: http://stackoverflow.com/questions/40881822/cannnot-focus-element-in-dropdown-test

Example of a violation:

```js
element(by.tagName('option[value="Test"]'))
```

Here, a CSS selector is used as a tag name locator value.

**Comment by alecxe** [comment]
@eLRuLL thanks so much for yet another PR!
